### DISCUSSION TITLE
Support New Lines in Text

### DISCUSSION THREAD

**Discussion by StormyIceLeopard** [streamdeck-linux-gui/streamdeck-linux-gui#31] [Ideas / Feature Requests]
I would like to be able to put text on multiple lines to better fit in the tiny button.
But pressing enter key does nothing.

**Comment by RemcoSchrijver** [comment]
Think this might be a good first issue so posting it there would be good.

**Comment by ajbucci** [comment]
I have a working implementation for this. Just need to read through the contribution guidelines before submitting a pull request.

**Comment by Danie10** [comment]
OK I see the dev release in AUR is showing this working quite nicely now, with buttons on the right to scroll vertically and horizontally for placement.
![Screenshot_20230822_145402](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/1153726/d527c623-9c76-45d7-86e6-8c013eb74e87)

**Reply by coolapso** [reply]
I am planing to merge to main and make stable release sometime this week or at latest during the weekend

**Comment by coolapso** [comment]
https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v3.1.0

This has been done and released on v3.1.0
